File: src/dto/presentazione.dto.ts


export class PresentazioneDTO{
  id: number;
  nome: string;
  dataCreazione: Date;
  ultimaModifica: Date;
  utente: number;

  constructor(id: number, utente: number, nome?: string, dataCreazione?: Date, ultimaModifica?: Date){
    this.id = id ;
    this.utente = utente;
    this.nome = nome ? nome : "";
    this.dataCreazione = dataCreazione ? dataCreazione : new Date();
    this.ultimaModifica = ultimaModifica ? ultimaModifica : new Date();
  }
}

File: Presentazione/Presentazione/DTO/PresentazioneDTO.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;


namespace Presentazione.Models
{
    
    public class PresentazioneDTO
    {

        public long id { get; set; }

        public string nome { get; set; }

        public DateTime dataCreazione { get; set; }

        public DateTime ultimaModifica { get; set; }

    }
}
